config/develop/s3-intermediate-bucket.yaml

```diff
@@ -0,0 +1,4 @@
+template_path: s3-bucket.yaml
```

Comment:
`template_path` is now deprecated.  Follow [this example](https://github.com/Sage-Bionetworks-Workflows/nextflow-infra/blob/c70701221a4045663a56e9fac6bb8735b1d51e3a/config/infra-prod/nextflow-vpc.yaml#L1-L3)

Reply:
Done